homeassistant/components/recorder/migration.py

```diff
@@ -1122,9 +1156,24 @@ def _apply_update(  # noqa: C901
             ("statistics", ("metadata_id",)),
             ("statistics_short_term", ("metadata_id",)),
         )
+        dropped_constraints = [
+            dropped_constraint
+            for table, columns in foreign_columns
+            for column in columns
+            for dropped_constraint in _drop_foreign_key_constraints(
+                session_maker, engine, table, column
+            )
+        ]
+        _LOGGER.debug(dropped_constraints)
```

Comment:
leftover debug or missing message?

Reply:
Intentially added, but I agree a message would be good 👍